contracts/JBETHPaymentTerminalStore.sol

```diff
@@ -370,10 +399,14 @@ contract JBETHPaymentTerminalStore {
       : PRBMathUD60x18.div(_amount, prices.priceFor(_currency, JBCurrencies.ETH));
 
     // The amount being distributed must be available.
-    require(distributedAmount <= balanceOf[_projectId], '0x40: INSUFFICIENT_FUNDS');
+    if (distributedAmount > balanceOf[_projectId]) {
+      revert JBErrors.INSUFFICIENT_FUNDS();
+    }
 
     // The amount being distributed must be at least as much as was expected.
-    require(_minReturnedWei <= distributedAmount, '0x41: INADEQUATE');
+    if (_minReturnedWei > distributedAmount) {
+      revert INADEQUATE();
```

Comment:
There should be separate errors for each of these cases. They were separated before by the hex number.  but the name should signify this in some way.